test/integration/targets/cloudscale_floating_ip/aliases

```diff
@@ -0,0 +1,2 @@
+cloud/cloudscale
+unsupported
```

Comment:
What is preventing these tests from being run in CI?

Reply:
Ah, never mind. I read this and thought cloudstack instead of cloudscale.

Reply:
We are working on it ;-) This is basically a prerequisite for this as it ensures that tests cleanup reasources regardless of success.